arwen/contexts/runtime.go

```diff
@@ -60,6 +61,8 @@ func (context *runtimeContext) InitState() {
 }
 
 func (context *runtimeContext) CreateWasmerInstance(contract []byte, gasLimit uint64) error {
+	defer arwen.TimeTrack(time.Now(), "CreateWasmerInstance")
```

Comment:
This will get called a lot... `TimeTrack` prints directly to `stdout`, consuming time and filling up the terminal. Do we want it here? It is useful for benchmarking, but not as a permanent feature.

Reply:
Not a permanent feature. This will write to Arwen's stdout, which is captured and displayed with log.Trace() at Node's side. Indeed, it will fill up the terminal during benchmarks. Removed it for now, we will add it again perhaps when Arwen will reference the "true" logger component.